pkg/hotplug-disk/hotplug-disk.go

```diff
@@ -76,6 +79,26 @@ func (h *hotplugDiskManager) GetHotplugTargetPodPathOnHost(virtlauncherPodUID ty
 	return "", fmt.Errorf("Unable to locate target path: %s", podpath)
 }
 
+// GetFileSystemDirectoryTargetPathFromHostView gets the directory path in the target pod (virt-launcher) on the host.
+func (h *hotplugDiskManager) GetFileSystemDirectoryTargetPathFromHostView(virtlauncherPodUID types.UID, volumeName string, create bool) (string, error) {
+	targetPath, err := h.GetHotplugTargetPodPathOnHost(virtlauncherPodUID)
+	if err != nil {
+		return "", err
+	}
+	directoryPath := filepath.Join(targetPath, volumeName)
+	exists, err := diskutils.FileExists(directoryPath)
+	if err != nil {
+		return "", err
+	}
+	if !exists && create {
+		err = os.Mkdir(directoryPath, 0750)
+		if err != nil {
+			return "", err
+		}
+	}
+	return directoryPath, err
```

Comment:
why err and not nil?

Reply:
This function is pretty much a copy of GetFileSystemDiskTargetPathFromHostView with a small change :) I can beautify it